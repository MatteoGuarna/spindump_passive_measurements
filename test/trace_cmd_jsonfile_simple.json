[
{ "Event": "new", "Type": "ICMP", "Addrs": ["31.133.149.35","212.16.98.51"], "Session": "48895", "Ts": 1553417873690045, "State": "Starting", "Notes": "No response", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["31.133.149.35","212.16.98.51"], "Session": "48895", "Ts": 1553417873728020, "State": "Up", "Notes": "No response", "Right_rtt": 37975, "Packets1": 1, "Packets2": 0, "Bytes1": 84, "Bytes2": 0, "Bandwidth1": 84, "Bandwidth2": 0 },
{ "Event": "packet", "Type": "ICMP", "Addrs": ["31.133.149.35","212.16.98.51"], "Session": "48895", "Ts": 1553417874694664, "State": "Up", "Dir": "initiator", "Length": 84, "Packets1": 2, "Packets2": 1, "Bytes1": 168, "Bytes2": 84, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["31.133.149.35","212.16.98.51"], "Session": "48895", "Ts": 1553417874732523, "State": "Up", "Right_rtt": 37859, "Packets1": 2, "Packets2": 1, "Bytes1": 168, "Bytes2": 84, "Bandwidth1": 84, "Bandwidth2": 84 }
]
