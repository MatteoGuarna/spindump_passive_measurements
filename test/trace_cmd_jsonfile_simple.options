--textual --json-input-file test/trace_cmd_jsonfile_simple.json
