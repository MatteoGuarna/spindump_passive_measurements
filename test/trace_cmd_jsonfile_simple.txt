Providing a JSON event file as input, containing a simple ICMP exchange. Should succeed.
