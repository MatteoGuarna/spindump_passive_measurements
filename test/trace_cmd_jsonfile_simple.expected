ICMP 31.133.149.35 <-> 212.16.98.51 65470 at 1553417873728020 measurement starting right 37975 packets 1 0 bytes 84 0 bandwidth 84 0
ICMP 31.133.149.35 <-> 212.16.98.51 65470 at 1553417874732523 measurement starting right 37859 packets 2 1 bytes 168 84 bandwidth 84 84
ICMP 31.133.149.35 <-> 212.16.98.51 65470 at delete starting packets 2 1 bytes 168 84 bandwidth 84 84
